### PR TITLE
Alix's Pull Request

### PR DESCRIPTION
**Pull request by AlixYAN**
Useless comment :)

### CODE REVIEW

**Comment by MKourdourli** [comment]
Are you sure Rainbow parrot can dance?!

File: alix_yan_file.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+*dancing rainbow parrot*
+
+First I was afraid, I was petrified
+Kept thinking I could never live without you by my side
+But then I spent so many nights
```
Comment (PabloRdrRbl) [inline alix_yan_file.txt:5]: Hola Alix. Look: I can write comments on individual lines to ask you to make modifications before finally accepting your review.

**Review by PabloRdrRbl — APPROVED** [review]
Now, I approve.

**Review by MKourdourli — APPROVED** [review]
I'am not a good singer but i'll approve this one !

**Pull request merged**